packages/gatsby-source-drupal/src/utils.js

```diff
@@ -468,7 +469,7 @@ export function drupalCreateNodeManifest({
       )
       warnOnceToUpgradeGatsby = true
     }
-    const manifestId = `${id}-${updatedAt}`
+    const manifestId = `${id}-${langcode}-${updatedAt}`
```

Comment:
Should we just reuse the logic here? https://github.com/gatsbyjs/gatsby/blob/63828fc109694427a57a18fef0f69767fd834028/packages/gatsby-source-drupal/src/normalize.js#L177-L205

Reply:
As we discussed there's a few differences with how we want to handle the langcode here vs the langcode we use to generate the node ID so it's probably better not to reuse this logic.